src/Domain/Entity/ContactMediaMap.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace BBC\ProgrammesPagesService\Domain\Entity;
+
+use BBC\ProgrammesPagesService\Domain\Enumeration\ContactMediaEnum;
+use BBC\ProgrammesPagesService\Domain\ValueObject\ContactMedia;
+use InvalidArgumentException;
+
+class ContactMediaMap
+{
+    /**
+     * Example:
+     *  [
+     *      'email'    => [<ContactMedia>, <ContactMedia>, <ContactMedia>],
+     *      'facebook' => [<ContactMedia>],
+     *  ]
+     *
+     * @var mixed[]
```

Comment:
`@var ContactMedia[][]`?

Reply:
I prefer to put an example. That kind of [][]....still is so blur...that for instance doesn't say anything about if we use numerical/string keys.

Reply:
I'm not saying to get rid of the example. Keep it if you like. Just change `@var mixed[]`